src/main/java/me/dinowernli/grpc/prometheus/MonitoringClientCall.java

```diff
@@ -16,6 +16,7 @@
   private final GrpcMethod grpcMethod;
   private final Configuration configuration;
   private final Clock clock;
+  private Metadata requestHeaders;
```

Comment:
Would it be clearer to call this requestMetadata, here an in other instance of fields and local variables?